nodes/textures/wireframe.py

```diff
@@ -0,0 +1,34 @@
+import bpy
+from bpy.props import EnumProperty, FloatProperty
+from ..base import LuxCoreNodeTexture
+from ... import utils
+from ...utils import node as utils_node
+
+
+class LuxCoreNodeTexWireframe(bpy.types.Node, LuxCoreNodeTexture):
+    bl_label = "Wireframe"
+    bl_width_default = 200
+
+    #width: FloatProperty(update=utils_node.force_viewport_update, name="Line Width", description="Width of line", min=0, subtype="DISTANCE", unit="LENGTH", default=0.1)
+    
+    def init(self, context):
+        self.add_input("LuxCoreSocketFloatPositive", "width", 0.1)
```

Comment:
The "width" property is not texturable in LuxCore, see https://github.com/LuxCoreRender/LuxCore/blob/ad76a75be398af2986d785dc61f0b95a19b8d31c/src/slg/scene/parsetextures.cpp#L606

So you can not use a socket for it, you'll have to use the FloatProperty.

Reply:
It may be very useful to make it texturable for "Dirty" texture (i.e. something I'm probably going to add). But it isn't for the moment.